[2025-03-13T20:50:33.011+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-13T20:50:33.026+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_edi.process_txt_to_postgres manual__2025-03-13T20:50:30.896041+00:00 [queued]>
[2025-03-13T20:50:33.034+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_edi.process_txt_to_postgres manual__2025-03-13T20:50:30.896041+00:00 [queued]>
[2025-03-13T20:50:33.035+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-13T20:50:33.046+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): process_txt_to_postgres> on 2025-03-13 20:50:30.896041+00:00
[2025-03-13T20:50:33.054+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=92) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-13T20:50:33.055+0000] {standard_task_runner.py:72} INFO - Started process 94 to run task
[2025-03-13T20:50:33.056+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pipeline_edi', 'process_txt_to_postgres', 'manual__2025-03-13T20:50:30.896041+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_edi.py', '--cfg-path', '/tmp/tmpp6ni7p35']
[2025-03-13T20:50:33.058+0000] {standard_task_runner.py:105} INFO - Job 11: Subtask process_txt_to_postgres
[2025-03-13T20:50:33.248+0000] {task_command.py:467} INFO - Running <TaskInstance: pipeline_edi.process_txt_to_postgres manual__2025-03-13T20:50:30.896041+00:00 [running]> on host 971f0f544d4a
[2025-03-13T20:50:33.325+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='user' AIRFLOW_CTX_DAG_ID='pipeline_edi' AIRFLOW_CTX_TASK_ID='process_txt_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T20:50:30.896041+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-13T20:50:30.896041+00:00'
[2025-03-13T20:50:33.326+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-13T20:50:33.349+0000] {logging_mixin.py:190} INFO - üöÄ Iniciando processamento do arquivo: /opt/***/dags/VENDASUN12032025140133.txt
[2025-03-13T20:50:33.349+0000] {logging_mixin.py:190} INFO - üïí Data/hora do arquivo extra√≠da: 2025-03-12 14:01:33
[2025-03-13T20:50:33.355+0000] {logging_mixin.py:190} INFO - üìÑ Arquivo lido com sucesso! Exemplo de dados:
[2025-03-13T20:50:33.367+0000] {logging_mixin.py:190} INFO -   tipo_registro cnpj_agente_distribuicao  ... cep_cliente tipo_documento
0             D           23637077001730  ...           1     N08740-270
1             D           23637077001730  ...           1     N07263-000
2             D           23637077001730  ...           1     N08557-505
3             D           23637077001730  ...           1     N07041-030
4             D           23637077001730  ...           1     N08740-270

[5 rows x 10 columns]
[2025-03-13T20:50:33.375+0000] {logging_mixin.py:190} INFO - üîç Contagem de valores nulos antes da limpeza:
[2025-03-13T20:50:33.377+0000] {logging_mixin.py:190} INFO - tipo_registro                 0
cnpj_agente_distribuicao      0
identificacao_cliente         0
data_transacao                0
numero_documento              0
codigo_produto                0
quantidade                    0
preco_venda                   0
cep_cliente                  45
tipo_documento                0
codigo_vendedor             363
dtype: int64
[2025-03-13T20:50:33.378+0000] {logging_mixin.py:190} INFO - ‚úÖ 363 registros v√°lidos ap√≥s a limpeza!
[2025-03-13T20:50:33.451+0000] {logging_mixin.py:190} INFO - ‚úÖ Dados inseridos com sucesso no banco!
[2025-03-13T20:50:33.451+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-13T20:50:33.460+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-13T20:50:33.460+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=pipeline_edi, task_id=process_txt_to_postgres, run_id=manual__2025-03-13T20:50:30.896041+00:00, execution_date=20250313T205030, start_date=20250313T205033, end_date=20250313T205033
[2025-03-13T20:50:33.510+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-13T20:50:33.526+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-13T20:50:33.527+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
